src/pagesComponents/Dashboard/Popup/Slide.js

```diff
@@ -0,0 +1,29 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+  ImagesWrapper, SlideParagraph, SlideTitle, SlideWrapper,
+} from '../styles';
+
+export default function Slide({
+  image1, image2, title, paragraph,
+}) {
+  return (
+    <SlideWrapper>
+      <ImagesWrapper className="images">
+        <img src={image1} alt="Popup" className="popup1" />
```

Comment:
Do not pass props like `image1` and `image2`, always be dynamic, pass an array of images instead.